### PR TITLE
Fe template refactor

### PR DESCRIPTION
**Pull request by bobbi-hbrown**
Replacement PR for https://github.com/communitiesuk/funding-service-design-post-award-submit/pull/84 due to tagging issues

### CODE REVIEW

**Review by Connor-OS — APPROVED** [review]
All good, once this has passed QA

File: app/templates/main/validation-errors.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+{% block beforeContent %}
+{{ super() }}
+{% endblock beforeContent %}
+
+{% block content %}
+    <div class="upload-data-container govuk-!-margin-top-5">
+        <form method="post" action="{{ url_for('main.upload') }}" enctype="multipart/form-data">
+            <h2 class="govuk-heading-m">There are errors in your return</h2>
+            <p class="govuk-body">Fix these errors and re-upload your return.</p>
```
Comment (Connor-OS) [inline app/templates/main/validation-errors.html:20]: This line is "The following errors have been found in your return." on main atm. The content here is what design are asking us to change too. I think the idea for this PR is to maintain all functionality from the current test env

**Review by Connor-OS — CHANGES_REQUESTED** [review]
Almost there

**Review by Connor-OS — APPROVED** [review]
Good to go 👍

**Comment by cyrusdobbs** [comment]
Can this be merged with squash when it's merged 👍

**Pull request merged**